src/Sql/Sql/help/Invoke-AzSqlInstanceExternalGovernanceStatusRefresh.md

```diff
@@ -12,9 +12,25 @@ Refreshes the value of external governance on the managed instance.
 
 ## SYNTAX
 
+### SetByNameParameterSet
```

Comment:
Here we should actually change to **Invoke**ByNameParameterSet because these examples are visible in documentation and in Help cmdlet